packages/date/src/DateRange.js

```diff
@@ -223,7 +223,9 @@ const DateRange = ({
 
                 // Focus the calendar icon once clicked because we don't
                 // want to get back in the loop of opening the calendar
-                calendarIconRef.current.parentElement.focus();
+                if (calendarIconRef && calendarIconRef.current) {
```

Comment:
I am unsure if this calendarIconRef is even used? Preference would be to delete it all together, but I am always seeing this ref as undefined. I don't see this ref being passed to the DateRangePicker or use anywhere else. The screen does not have any adverse behavior - just a console error.

Reply:
I think we can remove the ref completely. This was missed when we removed the ability to add a calendar icon

Reply:
Love it, I was hoping so! Doing that now